.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
```

Comment:
whats a rope project?

Reply:
Note sure, honestly. I added it to my .gitignore this last commit to keep it from cluttering my git staging process.

Reply:
I'd remove it from git altogether then

Reply:
I didn't realize the older version was already tracked. I'll remove it in the next commit.